apidoc/Titanium/UI/TableView.yml

```diff
@@ -1350,6 +1350,19 @@ properties:
     platforms: [iphone, ipad]
     availability: creation
 
+  - name: showSearchBarInNavigation
```

Comment:
Really quick and minor comment: Usually, the `UINavigationBar` is referenced as the `navBar` in Titanium, e.g. `navBarHidde` and `hideNavBar` / `showNavBar`. For the sake of parity, this could become `showSearchBarInNavBar` if you agree.

Reply:
Agree. Updated PR.